client/rpcserver/handlers.go

```diff
@@ -280,13 +277,21 @@ func ListCommands() string {
 	return s[:len(s)-1]
 }
 
-// CommandUsage returns a help message for cmd or an error if cmd is unknown.
-func CommandUsage(cmd string) (string, error) {
+// commandUsage returns a help message for cmd or an error if cmd is unknown.
+func commandUsage(cmd string, includePasswords bool) (string, error) {
 	msg, exists := helpMsgs[cmd]
 	if !exists {
-		return "", fmt.Errorf("%w: %s", ErrUnknownCmd, cmd)
+		return "", fmt.Errorf("%w: %s", errUnknownCmd, cmd)
 	}
-	return fmt.Sprintf("%s %s\n\n%s", cmd, msg[0], msg[1]), nil
+	// If help should include password arguments and this command has
+	// password arguments, return them as part of the help message.
+	if includePasswords && msg.pwArgsShort != "" {
+		return fmt.Sprintf("%s %s %s\n\n%s\n\n%s\n\n%s\n\n%s",
+			cmd, msg.pwArgsShort, msg.argsShort, msg.cmdSummary, msg.pwArgsLong,
+			msg.argsLong, msg.returns), nil
+	}
+	return fmt.Sprintf("%s %s\n\n%s\n\n%s\n\n%s", cmd, msg.argsShort, msg.cmdSummary,
+		msg.argsLong, msg.returns), nil
```

Comment:
I noticed that this adds a unnecessary empty lines when the `argsLong` is empty (`init`, `version`, `wallets`).